update_employees.py

```diff
@@ -270,11 +285,12 @@ def build_payload(
                 # if any of the fields differ, add banner record to payload
                 if is_different(r_hr, r_knack):
                     payload.append(r_hr)
-                    result["updates"].append((r_hr[name_field]))
```

Comment:
i removed these appends to `result` because they were falling out of sync with what records were actually updated due to filtering that happens later on in the script.